server_src/lib/serverCommands.js

```diff
@@ -119,10 +119,9 @@ class SystemCommands {
     await pullLatestRepository(force.forcePull);
 
     // Check to see if npm packages are missing and if so install them...
-    const missingPackages = await doWeHaveMissingPackages();
-
+    const missingPackagesList = await returnListOfMissingPackages();
     // If we have missing packages alert the user and wait for their response, if response given then install missing deps.
-    if (missingPackages) {
+    if (missingPackagesList.length > 0) {
```

Comment:
Falsy check?

Reply:
That's always going to return an array, empty or not.